src/pages/nft/class/_classId/index.vue

```diff
@@ -444,7 +444,6 @@ import crossSellMixin from '~/mixins/cross-sell';
 import navigationListenerMixin from '~/mixins/navigation-listener';
 import utmMixin from '~/mixins/utm';
 import alertMixin from '~/mixins/alert';
-import couponMixin from '~/mixins/coupon';
```

Comment:
Maybe we can keep all the coupon mixins, but make sure `getApplicableCoupon` return nothing for now?

Reply:
[🔥 Disable christmas coupon functionality](https://github.com/likecoin/liker-land/pull/2006/commits/e375bec7f6658092a538e7be58a96c1c1931d71f)